lib/uof_feed/handlers/data_schema_inspect.ex

```diff
@@ -0,0 +1,26 @@
+defmodule UofFeed.Handlers.DataSchemaInspect do
+  @moduledoc """
+  Process incoming messages from XML into Elixir structures and log the result
+
+  Processing is done via `UofFeed.Mapper.call/1`.
+  """
+  @behaviour UofFeed.Handlers.Behaviour
```

Comment:
I'd rename `UofFeed.Handlers.Behaviour` to just `UofFeed.Handler` to avoid the redundancy and extra keystrokes. I don't think it leads to any loss of clarity. Then all the provided impls should be renamed to `UofFeed.Handler.*` instead of `UofFeed.Handlers.*` for consistency. WDYT?

```suggestion
  @behaviour UofFeed.Handler
```

Reply:
yes, after going through the code I agree with the suggestions, I have adjusted the naming